components/formikFormControllers/Dropdown/Dropdown.tsx

```diff
@@ -18,6 +18,7 @@ type Props<FormValuesType> = {
   formik: FormikProps<FormValuesType>;
   size?: "medium" | "small";
   isCombobox?: boolean;
+  disabled?: boolean;
```

Comment:
Can use isDisabled instead to keep it consistent with other booleans in the codebase

Reply:
Okay sure, I used `disabled` because the underlying component uses `disabled` too but yeah it's better to be consistent